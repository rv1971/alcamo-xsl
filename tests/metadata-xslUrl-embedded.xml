<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/metadata.xsl"/>

    <xsl:template match="/">
      <a:result>
        <a:item>
          <xsl:value-of select="$a:xslUrl"/>
        </a:item>

        <a:item>
          <xsl:value-of select="$a:xslDirUrl"/>
        </a:item>
     </a:result>
    </xsl:template>
  </xsl:stylesheet>
</a:doc>
