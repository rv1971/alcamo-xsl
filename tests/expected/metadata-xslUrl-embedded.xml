<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>#xslt</a:item>
  <a:item/>
</a:result>
